### DISCUSSION TITLE
Formulario

### DISCUSSION THREAD

**Discussion by CarlosToapantaN** [LaunchX-InnovaccionVirtual/CursoIntroPython#332] [Q&A]
Hola. Quería llenar el formulario pero me dice que ya no acepta respuestas. Por favor ayuda.

**Comment by Megaman8** [comment]
Igual me manda el error de que ya no se aceptan respuestas :(

**Comment by dany6x** [comment]
También a mi

**Reply by christoferguzman** [reply]
yo tampoco y en el live mencionaron que era hasta el lunes 21 a las 8:00 pm

**Comment by antonietamg** [comment]
Estoy en la misma situación

![Screen Shot 2022-02-20 at 8 31 44 PM](https://user-images.githubusercontent.com/62116253/154879465-1d8473be-6f12-42ca-98ec-c6e261e7ec32.png)

**Comment by ApablazaYevenes** [comment]
Saludos. Tengo el mismo problema

**Comment by Gustavo-Valdez** [comment]
Mismo problema

**Comment by antonietamg** [comment]
Ya encontré el error. Cierren su sesión de Ofice 365. Sálganse por completo haciendo click en Sign Out.
Ya que estén afuera, vayan a este link: https://forms.office.com/r/mLLd3dbfPX

**Reply by Gustavo-Valdez** [reply]
Ya lo abrieron de nuevo

**Reply by antonietamg** [reply]
Ahhh perfecto!!! Entonces fue coincidencia jejeje. Qué alivio que ya lo abrieron otra vez.

**Comment by Megaman8** [comment]
En raliada ya reactivaron el envío del formulario!!!

**Comment by MezaDonovan** [comment]
Yo lo solucione entrado en una pestaña en modo incógnito! Inténtenlo!
